client/state/ui/guided-tours/selectors.js

```diff
@@ -123,8 +129,17 @@ const findTriggeredTour = state => {
 	} );
 };
 
+const shouldBail = state => {
+	// bail if we're on a blacklisted page
```

Comment:
I have nothing against returns like this, but I've seen we tend to simplify these to just one statement: `return includes(…);`